koji_containerbuild/plugins/hub_containerbuild.py

```diff
@@ -53,6 +58,30 @@ def _get_task_opts_and_opts(opts, priority, channel):
 
     return opts, taskOpts
 
+
+def _check_priority_allowlist(target, priority):
+    CONFIG = koji.read_config_files([(CONFIG_FILE, True)])
+    priority_target_list = CONFIG.get('priority_allowlist', 'priority_target_list_url')
+    req = Request(priority_target_list)
+    try:
+        response = urlopen(req)
+        prio_target_list = yaml.safe_load(response.read().decode())
+        if target in prio_target_list['targets']:
+            if not priority:
+                priority = koji.PRIO_DEFAULT - 1
+            else:
+                priority -= 1
```

Comment:
What happen when priority is 0 or -1 ?

Reply:
AIUI, both are valid; "only admins have the right to specify a negative priority"